types/pretty-ms/index.d.ts

```diff
@@ -14,6 +15,9 @@ declare namespace prettyMs {
         msDecimalDigits?: number;
```

Comment:
[secDecimalDigits](https://github.com/sindresorhus/pretty-ms#secdecimaldigits) option seems to be missing here.

Reply:
This option is one line above the one you have commented on.

Reply:
Well, seems like I'm blind.